package.json

```diff
@@ -164,6 +164,7 @@
     "@openforis/arena-core": "^0.0.79",
     "@reduxjs/toolkit": "^1.8.1",
     "@socket.io/redis-streams-adapter": "^0.1.0",
+    "@types/csv-write-stream": "^2.0.3",
```

Comment:
We are using [fast-csv](https://www.npmjs.com/package/fast-csv) in a couple of places in the codebase. Since it works similarly, I'll switch to it here instead of csv-write-stream. What do you think?